nautobot/extras/templates/extras/exporttemplate.html

```diff
@@ -1,57 +1,11 @@
-{% extends 'base.html' %}
+{% extends 'generic/object_detail.html' %}
 {% load buttons %}
 {% load custom_links %}
 {% load helpers %}
 {% load plugins %}
-{% load static %}
-{% load tz %}
 
-{% block header %}
-    <div class="row noprint">
-        <div class="col-sm-8 col-md-9">
-            <ol class="breadcrumb">
-                <li><a href="{% url 'extras:exporttemplate_list' %}">Export Templates</a></li>
-                {% if object.name %}
-                    <li><a href="{% url 'extras:exporttemplate' pk=object.pk %}">{{ object.name }}</a></li>
-                {% endif %}
-            </ol>
-        </div>
-    </div>
-
-    <div class="pull-right noprint">
-        {% plugin_buttons object %}
-        {% if perms.extras.add_exporttemplate %}
-            {% clone_button object %}
-        {% endif %}
-        {% if perms.extras.change_exporttemplate %}
-            {% edit_button object %}
-        {% endif %}
-        {% if perms.extras.delete_exporttemplate %}
-            {% delete_button object %}
-        {% endif %}
-    </div>
-
-    <h1>{% block title %}{{ object }}{% endblock %}</h1>
-    {% include 'inc/created_updated.html' %}
-    <div class="pull-right noprint">
-        {% custom_links object %}
-    </div>
-
-    <ul class="nav nav-tabs">
-        <li role="presentation"{% if not active_tab %} class="active"{% endif %}>
-            <a href="{{ object.get_absolute_url }}">Export Templates</a>
-        </li>
-        {% if perms.extras.view_objectchange %}
-            <li role="presentation"{% if active_tab == 'changelog' %} class="active"{% endif %}>
-                <a href="{% url 'extras:exporttemplate_changelog' pk=object.pk %}">Change Log</a>
-            </li>
-        {% endif %}
-    </ul>
-
-{% endblock %}
 {% block content %}
```

Comment:
Wonder if this page would be better handled as left-column "Details", right-column "Template", and full-width "Code Template"?

Reply:
Great catch, thanks! This was one of the very first ones I refactored and didn't circle back.